heron/stmgr/src/cpp/server/stmgr-main.cpp

```diff
@@ -55,6 +56,8 @@ int main(int argc, char* argv[]) {
   sp_int32 shell_port = atoi(argv[11]);
   sp_string heron_internals_config_filename = argv[12];
 
+  HeapProfilerStart("stmgr");
```

Comment:
The profiling is enabled during the whole stmgr life time.

What about adding two handlers to the stmgr server to control the profiling start & end:
1. HandleStartHeapProfilingMessage: HeapProfilerStart()
2. HandleStopHeapProfilingMessage: HeapProfilerStop()
In this way, the profiling is not enabled by default and there is no impact on performance.